tests/cms/views/pages/test_page_model.py

```diff
@@ -0,0 +1,76 @@
+import pytest
```

Comment:
I am not convinced on the file name. If this file should generally contain tests for the page model, shouldn't it be in the `tests/cms/models/pages` directory?

Reply:
Do you suggest changing the directory or changing the name of the file? :)

Reply:
hmm, maybe both? Probably depends on what kind of tests this file should contain in the future. If its only about testing the clean_content functionality, I'd move this test to `tests/cms/utils/test_content_utils.py`.

Reply:
Yes, makes sense!